README.md

```diff
@@ -228,6 +228,7 @@ If you use the package, please cite:
   author = {M. J. Muckley and R. Stern and T. Murrell and F. Knoll},
   title = {{TorchKbNufft}: A High-Level, Hardware-Agnostic Non-Uniform Fast {Fourier} Transform},
   booktitle = {ISMRM Workshop on Data Sampling \& Image Reconstruction},
-  year = 2020
+  year = 2020,
+  url = {https://github.com/mmuckley/torchkbnufft}
```

Comment:
```suggestion
  note = {Source code available at https://github.com/mmuckley/torchkbnufft}.
```

Reply:
Looks good to me. The Workshop posters are unfortunately not available, even after an embargo, except to that year's attendees.